run-ib-pr-matrix.sh

```diff
@@ -86,11 +86,13 @@ pushd "$WORKSPACE/matrix-results"
     LOG=runall-report-step123-.${BUILD_ID}.log
     for WF in $(grep -a '^[1-9][0-9]*' ${LOG} | grep ' Step[0-9]' | sed 's| .*||' | sort | uniq ) ; do
       memFile=maxmem_profile_$(echo $WF | cut -d_ -f1).txt
+      memFileName=maxmem_profile_$(echo $WF).txt
```

Comment:
@smuzaffar I make a copy of the file here.

Reply:
Do you need this copy? why not file with workflow number enough?